### ISSUE TITLE
Create Kotlin project fail due to pom.xml-template.ftl

### ISSUE THREAD

**Issue by sbert** [quarkusio/quarkus#3541]
**Describe the bug**
The command 
```
mvn io.quarkus:quarkus-maven-plugin:0.21.1:create \
    -DprojectGroupId=org.acme \
    -DprojectArtifactId=rest-kotlin \
    -DclassName="org.acme.rest.GreetingResource" \
    -Dpath="/greeting" \
    -Dextensions="kotlin,resteasy-jsonb"
```

failed due to : 
`[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:0.21.1:create (default-cli) on project standalone-pom: Template resource is missing: /templates/basic-rest/kotlin/pom.xml-template.ftl -> [Help 1]`

**Environment (please complete the following information):**
 - Quarkus version or git rev: 0.21.0 and 0.21.1

**Comment by gwenneg** [comment]
Hi @sbert and thanks for reporting this issue!

Would you like to contribute to Quarkus and submit a pull request to fix it?

**Comment by geoand** [comment]
I linked to #3542 which addresses the issue

**Comment by gwenneg** [comment]
Fixed in #3399.
